scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOPredictionAnomaly.scala

```diff
@@ -25,6 +25,8 @@ import org.apache.spark.sql.{Column, Row}
 trait H2OMOJOPredictionAnomaly extends H2OMOJOPredictionUtils {
   self: H2OMOJOModel =>
   def getAnomalyPredictionUDF(): UserDefinedFunction = {
+    logWarning("Starting from next major release, the content of 'prediction' column will be generated to " +
```

Comment:
nit: `next major release` -> `the next major release` in all warnings

Reply:
thanks!